src/formio/components/Children/Children.scss

```
@@ -0,0 +1,22 @@
+.openforms-children {
+  padding: 1em;
```

our spacing values are typically multiples of `4px` - but this should definitely be a design token:


```suggestion
  padding: var(--of-children-padding, 8px);
```
so that in different themes (and viewports), the padding can be modified.